shared/src/convention/convention.ts

```diff
@@ -272,7 +273,10 @@ export const flatParamsToGetConventionsForAgencyUserParams = (
         to: dateSubmissionTo,
       },
     },
-    sortBy,
+    sort: {
+      sortBy,
+      sortOrder,
+    },
```

Comment:
```suggestion
    sort: {
      by,
      order,
    },
```

Mais c'est une question de goût